content/en/docs/devops-user-guide/examples/a-maven-project.md

```diff
@@ -133,7 +133,13 @@ In this example, all workloads are deployed in `kubesphere-sample-dev`. You must
    
            stage('deploy to dev') {
              steps {
-               kubernetesDeploy(configs: 'deploy/dev-ol/**', enableConfigSubstitution: true, kubeconfigId: "$KUBECONFIG_CREDENTIAL_ID")
+                withCredentials([
+                    kubeconfigFile(
+                    credentialsId: env.KUBECONFIG_CREDENTIAL_ID,
+                    variable: 'KUBECONFIG')
+                    ]) {
+                    sh 'envsubst < devops-go-sample/deploy/all-in-one/devops-sample.yaml | kubectl apply -f -'
```

Comment:
Please confirm that file `devops-go-sample/deploy/all-in-one/devops-sample.yaml` is existing in repository <https://github.com/kubesphere/devops-maven-sample>.